lib/wallaby/browser.ex

```diff
@@ -857,6 +856,16 @@ defmodule Wallaby.Browser do
     end
   end
 
+  defp do_get(query, elements) do
+    getNumber = query.conditions[:get]
```

Comment:
Also I think it'd be better if we use the `get_number` method from the Query module here, to not bind this to the implementation details of where this comes from :)